src/App.tsx

```
@@ -56,9 +56,7 @@ const OrganizationFunds = lazy(
 );
 const OrganizationTransactions = lazy(
   () =>
-    import(
-      'screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'
-    ),
+    import('screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'),
```

ESLint

[error] Replace `'screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'` with `⏎······'screens/AdminPortal/OrganizationTransactions/OrganizationTransactions'⏎····`

(prettier/prettier)